lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -438,11 +438,13 @@ interface SendDestination {
 
 dictionary PrepareSendResponse {
     SendDestination destination;
-    u64 fees_sat;
+    u64? fees_sat;
+    f64? fees;
```

Comment:
Do you think it makes sense to rename this to `asset_fees` and in SendPaymentRequest `use_asset_fees` ?

Reply:
Done [29b8e3d](https://github.com/breez/breez-sdk-liquid/pull/779/commits/29b8e3d8fb9ffc055d807560ebcb92011bec9546)